Front-end-Projects/Intermediate/Product-Landing-Page/README.md

```diff
@@ -0,0 +1,67 @@
+<h1 align='center'><b>💥 Product-Landing-Page💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+
+
+<div align='center'>
+
+  ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)
+  ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white)
+
+</div>
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Description 📃
+
+<div>
+  <p>This project is a simple HTML and CSS implementation of a Product Landing Page. It aims to showcase the product in front of the customer in other to get maximum number of user engagement. This project Showcases the power of HTML and CSS.</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: How to run it? 🕹️
+
+<div>
+  <p>To run this project locally, follow these steps:
+
+1. Fork the repository.
+
+2. Clone the repository to your local machine:
+    git clone https://github.com/your-username/Product-Landing-Page.git
+
+3. Open the project folder in your preferred code editor, now you can view website in live.
+
+</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+<img src="screenshot.webp">
+<img src="screenshot2.webp">
+<img src="screenshot3.webp">
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
```

Comment:
Create a new line to avoid this link to visible.
![image](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/01c46244-a969-49bb-a2ff-9cfef1a92607)